src/EditorFeatures/Core.Wpf/SymbolSearch/SymbolSearchUpdateEngine.cs

```diff
@@ -31,14 +31,6 @@ internal partial class SymbolSearchUpdateEngine : ISymbolSearchUpdateEngine
         public SymbolSearchUpdateEngine(
             ISymbolSearchLogService logService,
             ISymbolSearchProgressService progressService)
-            : this(logService, progressService, CancellationToken.None)
```

Comment:
this is the constructor called for the OOP case (i.e. the non-test case).  As you can see, it always passes along CancellationToken.None.